lib/unifex/specs.ex

```diff
@@ -139,7 +139,14 @@ defmodule Unifex.Specs do
     `void on_unload(UnifexEnv *env, void * priv_data)`
 
   """
-  defmacro callback(hook, fun) when hook in [:load, :upgrade, :unload] and is_atom(fun) do
+  defmacro callback(hook, fun \\ nil) when hook in [:load, :upgrade, :unload] and is_atom(fun) do
+    fun =
+      if fun == nil do
```

Comment:
I think that creating a separate one-parameter definition invoking the current one would look better